src/box/lua/tuple_format.c

```diff
@@ -57,9 +57,8 @@ lbox_tuple_format_gc(struct lua_State *L)
 static int
 lbox_tuple_format_new(struct lua_State *L)
 {
-	int top = lua_gettop(L);
-	(void)top;
-	assert((1 <= top && 2 >= top) && lua_istable(L, 1));
+	if (!lua_istable(L, 1))
+		return luaL_error(L, "format should be a table");
```

Comment:
Event this should not be triggered normally we now use box error instead of Lua. I suggest to introduce `luaT_checktable` (see `luaT_checklstring` for example) and use it. In this case we can use `assert_error_msg_equals` for assertions in the test.

Reply:
Changed to `luaT_checktype(L, 1, LUA_TTABLE);`.